src/app/organizations/organization-details/manage-projects/manage-single-project/manage-single-project.component.scss

```diff
@@ -157,6 +157,7 @@
               }
               div {
                 color: $icons-color;
+                cursor: pointer;
```

Comment:
If something is clickable, it should be either a button or anchor link ("a" tag).

It's good to follow this: If something is clickable, and it's not a button, wrap it in anchor tag, set empty routerLink and set click listener:

```
<a [routerLink]="" (click)="openBeer($event)">
  <div>Anything clickable</div>
</a>
```

This way, this element will be clickable, visible in text readers, and have a pointer as a cursor on the desktop.